core/src/main/resources/jenkins/management/Messages.properties

```diff
@@ -22,7 +22,7 @@
 # THE SOFTWARE.
 #
 
-ConfigureLink.DisplayName=Configure System
+ConfigureLink.DisplayName=System Configuration
```

Comment:
Plenty of documentation will become outdated with this change.

Reply:
OK, I can keep the text intact for users with the `ADMINISTER` permission